stylesheets/base/_form.scss

```diff
@@ -0,0 +1,55 @@
+.form {
+    
+
+  @media screen and (min-width: $breakpoint-tablet) {
+    font-size: 1.5rem;
+  }
+
+  &__label {
+    color: #464343;

```

Comment:
I notice that another file adds sass variables for color, but in this one you don't use them.

Reply:
Its a autogenerated file, if we delete it, it reapears